test/genbank_importer_test.py

```diff
@@ -144,16 +145,37 @@ def test_genbank_to_genome(self, download_staging_file, update_staging_service):
 
         gbk_path = 'small_genbank.gbff'
         ws_obj_name = 'MyGenome'
+        expected_scientific_name = 'Escherichia coli str. K-12 substr. MG1655'
 
         params = {
           'staging_file_subdir_path': gbk_path,
           'genome_name': ws_obj_name,
+          'ncbi_taxon_id': '28077',
+          'relation_engine_timestamp_ms': 1625695904755,
```

Comment:
you can just pass 
`'taxon_id': '28077',
'time_stamp': 1625695904755,`
I don't see why you need the additional assignment `params['taxon_id'] = params['ncbi_taxon_id']`

Reply:
params['taxon_id'] = params['ncbi_taxon_id'] is needed because GFU is expecting params['taxon_id']